src/parser/core/modules/Statistics/TableStatistic.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import {AbstractStatistic, AbstractStatisticOptions} from './AbstractStatistic'
+import styles from './TableStatistic.module.css'
+
+interface FixedLengthArray<T, L extends number> extends ReadonlyArray<T> {
+	0: T
+	length: L
+}
```

Comment:
This is fully lifted from `PieChartStatistic`, not sure what the right approach is for sharing this code.